libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/entitlements/FilesEntitlement.java

```diff
@@ -239,9 +239,7 @@ public Stream<Path> resolveRelativePaths(PathLookup pathLookup) {
                 String path = pathLookup.settingResolver().apply(setting);
                 result = path == null ? Stream.of() : Stream.of(path);
             }
-            if (ignoreUrl) {
-                result = result.filter(s -> s.toLowerCase(Locale.ROOT).startsWith("https://") == false);
-            }
+            result = result.filter(s -> s.toLowerCase(Locale.ROOT).startsWith("https://") == false);
```

Comment:
We can't rely on all setting rejecting `http`, right? Shouldn't we make sure to exclude such urls as well, so we keep the filtetree clean?